features/account/customer_account/changing_password.feature

```diff
@@ -6,12 +6,12 @@ Feature: Changing a customer password
 
     Background:
         Given the store operates on a single channel in "United States"
-        And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and a password "whitehouse"
+        And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and a password "sylius"
         And I am logged in as "francis@underwood.com"
 
-    @ui
+    @ui @shop_user_account_api
```

Comment:
```suggestion
    @ui @api
```

Reply:
With just @api tag , the behat steps are duplicating ( same is on Customer and AccountContext)